tilt_modules/cert_manager/Tiltfile

```diff
@@ -47,8 +47,13 @@ def deploy_cert_manager(registry="quay.io/jetstack", version="v1.3.1", load_to_k
     # NOTE!
     # Applying the same manifest twice to same cluster kubectl get stuck with older versions of kubernetes/cert-manager.
     # https://github.com/jetstack/cert-manager/issues/3121
-    print("Installing cert-manager")
-    local("kubectl apply -f https://github.com/jetstack/cert-manager/releases/download/{}/cert-manager.yaml".format(version), quiet=silent, echo_off=silent)
+
+    #verify that the installed version is the same as the version parameter and cert-manager is installed
+    #install the cert-manager if one of these is false
+    installed_version = local("kubectl get deployment -n cert-manager cert-manager -o=jsonpath='{.spec.template.spec.containers[0].image}' | awk -F':' '{print $2}'")
```

Comment:
Does this `awk -F` command work the same on Linuxes and macOS? I ask because often there are BSD vs. GNU-type differences in these commands.

Also, could you squash the commits so there's only one?

Reply:
The commits should be squashed now - for the awk command, I double checked in a Linux container (my dev machine is a Mac) and it looks to work the same: 
`root@2c937a6f4520:/# uname -a
Linux 2c937a6f4520 6.5.11-linuxkit #1 SMP PREEMPT Wed Dec  6 17:08:31 UTC 2023 aarch64 aarch64 aarch64 GNU/Linux
root@2c937a6f4520:/# echo "quay.io/jetstack/cert-manager-controller:v1.3.1" | awk -F':' '{print $2}'
v1.3.1`